pkg/kubeconfig/create_kubecfg.go

```diff
@@ -152,6 +150,18 @@ func BuildKubecfg(cluster *kops.Cluster, keyStore fi.Keystore, secretStore fi.Se
 		}
 	}
 
+	if useKopsAuthenticationPlugin {
+		b.AuthenticationExec = []string{
+			"kops",
+			"helpers",
+			"kubectl-auth",
+			"--cluster=" + clusterName,
+			"--state=" + kopsStateStore,
+			"--lifetime=1h",
+			"--api-version=" + "v1beta1",
```

Comment:
Perhaps we should let lifetime and api-version default instead of freezing these values in the user's kubeconfig.

Reply:
The api-version is an internal detail I think, at least until kubectl introduces v1.  I would hope that would be a gradual transition and not something we need to bother the user with.

For the lifetime, my plan is to get this in there and then look at all the changes we've made.  For example - should this be the same as the admin lifetime?  We've taken a usability hit by not exporting the kubecfg, because it breaks the "infer the current cluster" experience.

I moved the variables to the top of the block and added a comment and a TODO.

Reply:
If we write them to the kubecfg then they will stay fixed for that user unless/until that user takes an explicit action to update them. If we omit the flags from the kubecfg and let cobra default them then upon kops upgrade any new default values will automatically take effect.

Reply:
Oh I get it now!  That's a good point :-)   I'll remove the defaults!